src/HomePage/NavBar/NavBar.scss

```diff
@@ -10,22 +10,28 @@
 }
 
 .navbar {
-  margin: 10px;
+  margin: 5px auto;
   display: flex;
-  justify-content: space-between;
-}
+  flex-direction: row;
```

Comment:
Is this needed? I believe the default direction is row

Reply:
Have removed